lib/puppet/type/service.rb

```diff
@@ -73,6 +73,11 @@ def retrieve
         provider.enabled?
       end
 
+      #This only works on Windows systems.
+      newvalue(:delayed, :event => :service_delayed_start) do
```

Comment:
Can you add a friendly Puppet error here: https://github.com/puppetlabs/puppet/pull/7702/files#diff-e021bc421d473ca033d243b6fd471d70R93 that: _Setting enable to delayed is only supported on Microsoft Windows_